pkg/networkservice/common/createpod/server_test.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2021 Doc.ai and/or its affiliates.
```

Comment:
Please add test cases with multiple requests for the same node:
1. When 1 pod is alive.
2. When 1 pod has already died.